project/pom.xml

```diff
@@ -217,12 +227,12 @@
     <project.build.outputTimestamp>1668154596</project.build.outputTimestamp>
 
     <!-- Plugin versions -->
-    <bnd.version>5.2.0</bnd.version>
+    <bnd.version>7.0.0-SNAPSHOT</bnd.version>
```

Comment:
I don't understand why we are using bnd in the first place -- is this some kind of OSGi thing?  @jbonofre

Reply:
Alternatively we could remove all uses of bnd and `<manifestFile>` from maven-jar-plugin.